### ISSUE TITLE
UI going crazy / flashing in recent release

### ISSUE THREAD

**Issue by hyperknot** [DamnWidget/anaconda#629]
## Expected Behaviour
UI should just work normally.

## Actual Behaviour
While typing, the UI is usually going crazy in recent release. It's flashing useless information, making it impossible to work productively.
Made a screen capture about it (in the video, I'm not pressing any other button except for the ones visible).
https://youtu.be/6feR38wh3rs

## Steps to Reproduce
Just start typing in a normal project.

## ST3, Anaconda and OS versions
ST: 3126
Anaconda: 2.1.16
OS X: 10.11.6

## ST3 Console Logs
https://bpaste.net/show/e96186c284d3

## Anaconda's JsonServer Logs
https://bpaste.net/show/309e94d360e9

**Comment by DamnWidget** [comment]
That just looks like the signatures being interfered by other plugin and making it to flicker. Which other plugins do you have installed alongside anaconda?

You can disable the signatures setting `enable_signatures_tooltip` as `false` in your configuration.

**Comment by hyperknot** [comment]
Here are my packages:

```
"installed_packages":
	[
		"Anaconda",
		"Clickable URLs",
		"Compare Side-By-Side",
		"Dockerfile Syntax Highlighting",
		"Filter Lines",
		"HexViewer",
		"INI",
		"JavaScriptNext - ES6 Syntax",
		"Jinja2",
		"LESS",
		"Neon Color Scheme",
		"Package Control",
		"Predawn",
		"Pretty JSON",
		"Python Flake8 Lint",
		"Python Improved",
		"Sass",
		"Schemr",
		"SCSS",
		"SideBarEnhancements",
		"StandardFormat",
		"SublimeLinter",
		"SublimeLinter-contrib-standard",
		"SublimeLinter-json",
		"SublimeLinter-shellcheck",
		"Themr",
		"URLEncode"
	]
```

But this one is something really new, I'll try to test it with some older Anaconda version.
Actually the most annoying part is not just the visual flickering, but that it actually inserts the automatic help text a few seconds after I started typing! So I end up with thing like `os.path.isfile(MYFILENAMEpath)`, just using capital letters to show the bits I typed.

**Comment by hyperknot** [comment]
So far, I've changed my Anaconda dir to be git controlled and added it to package control. Now the issue is not present, even if I go back to 2.1.16, or at least I cannot reproduce it.

However there is an even more serious issue. I've quit Sublime Text (possibly multiple times) and a python process eating 99% of my CPU, using 1 GB of memory each has stayed running. From the open file handles it is exactly the Anaconda server, as it has an open handle to the Anaconda log.

<img width="566" alt="screen shot 2017-03-19 at 01 19 37" src="https://cloud.githubusercontent.com/assets/494223/24077099/5ca084ba-0c42-11e7-948a-f4622badef29.png">
<img width="609" alt="screen shot 2017-03-19 at 01 19 27" src="https://cloud.githubusercontent.com/assets/494223/24077100/5ca09dec-0c42-11e7-8500-a3a36898fc76.png">

**Comment by DamnWidget** [comment]
Can you reproduce the hanging processes just restarting your ST3 in a consistent way? I couldn't reproduce any of your problems.

In fact anaconda does not insert any text shown in the tooltips that is a feature that does not exist in the plugin.

**Comment by hyperknot** [comment]
Do you mean that Anaconda isn't inserting `path` in `os.path.isfile(path)`? It definitely does it. If I disable it won't happen, if I enabled it does.

I'll look into how to reproduce it consistently, I couldn't do it either.

**Comment by DamnWidget** [comment]
What I mean is that anaconda just inserts any code in autocompletion events and that is.

**Comment by hyperknot** [comment]
Yes, normally it just inserts the code and selects it. But in that buggy / laggy version interaction it was inserting it about a second or two after I started typing, thus making the text actually appear.

I'm opening a separate ticket about 100% CPU in 2.1.17. It's not present in 2.1.16.

**Comment by DamnWidget** [comment]
It would be very useful to know if someone else is having problems with Jedi in 2.1.17 as I updated it, so probably there is a problem with it. Tells Package Control to not update anaconda at use previous version for now w

**Comment by hyperknot** [comment]
@DamnWidget I've opened a separate ticket for 2.1.17 as it's a totally unrelated issue I believe.
https://github.com/DamnWidget/anaconda/issues/632

**Comment by paregorios** [comment]
I'm seeing the same behavior as described at the top of this ticket (ST 3.0 build 3143)

```
        "installed_packages":
        [
                "3D Tool (Async Burst-Mode)",
                "A File Icon",
                "Anaconda",
                "Boxy Theme",
                "ColorSchemeEditor",
                "Colorsublime",
                "HTML5",
                "LaTeX Word Count",
                "Line Endings Unify",
                "Markdown Preview",
                "MarkdownEditing",
                "Monokai Extended",
                "Monokai JSON+",
                "Package Control",
                "PackageResourceViewer",
                "Pandoc Academic",
                "Pretty JSON",
                "RestructuredText Improved",
                "SublimeLinter",
                "SublimeLinter-pep8",
                "SublimeLinter-pyflakes",
                "Sunrise Theme",
                "Theme - Autumn",
                "Theme - Flatland",
                "Theme - Kronuz",
                "Theme - Lanzhou",
                "Theme - Soda",
                "Theme - Soda SolarizedDark",
                "Theme - SoDaReloaded",
                "WordCount"
        ]
```
